src/main/java/com/pusher/rest/Pusher.java

```diff
@@ -99,8 +103,34 @@ public Pusher(final String appId, final String key, final String secret) {
         this.appId = appId;
         this.key = key;
         this.secret = secret;
+        this.encryptionMasterKey = null;
+        configure();
+    }
```

Comment:
You could reduce some duplication by defining this in terms of the constructor which takes the encryption master key, e.g. something like

```java
public Pusher(final String appId, final String key, final String secret) {
    this(appId, key, secret, null);
}
```

But this is non-essential.